### ISSUE TITLE
Key Error shortcode_media not find

### ISSUE THREAD

**Issue by kharazian** [InstaPy/InstaPy#6381]
## Expected Behavior
at like_util.py the get_additional_data function get some post info which one of them is shortcode_media.

## Current Behavior
the shortcode_media key not found in the post_page (post_page = get_additional_data(browser))

**Comment by kazeox** [comment]
Hello, any update ? I've got the same issue.

**Comment by kharazian** [comment]
> Hello, any update ? I've got the same issue.

no, I faced problem. I made some changes to work to somebody solve the problem

**Comment by kazeox** [comment]
Now it work for me... I just uninstalled and reinstalled firefox, updated geckodriver.

**Comment by kharazian** [comment]
> Now it work for me... I just uninstalled and reinstalled firefox, updated geckodriver.

Hi, is your code working?

**Comment by kazeox** [comment]
Yes, more or less. Sometimes I have to re-launch it because a webpage didn't load correctly and the bot does not refresh the page automatically, it just quit after 300s. But most of the time "like by tag" , "interact with users" and "unfollow" work well. ( Sorry for my bad english i'm not a english-speaker)
